Client/Extensions/UIViewExtensions.swift

```diff
@@ -102,6 +102,26 @@ extension UIView {
         return spacer
     }
     
+    /// Adding multiple subviews in one instruction.
+    @discardableResult
+    func addSubviews(_ listofViews: [UIView]) -> Self {
+        for view in listofViews {
+            self.addSubview(view)
+        }
+        return self
+    }
+
+    /// List of subviews owned by the the view
+    public func subviews(where predicate: (_ view: UIView) -> Bool) -> [UIView] {
```

Comment:
It should be documented in the name or comment that this is a recursive method

Reply:
It is added inside the comment